app/request_handler.py

```diff
@@ -184,31 +184,35 @@ def _guessYelpId(placeName, lat, lon):
         return cachedId
 
     opts = {
-      # 'term': placeName,
+      'term': placeName,
       'limit': 20,
-      'radius_filter': 1000,
+      'radius_filter': 100,
```

Comment:
Remove `radius_filter` and `term`. Please see my previous comments #68 and now in #70.